appserver/tests/payara-samples/samples/clustered-singleton/clustered-singleton-test/src/main/java/fish/payara/samples/clustered/singleton/ClusteredAnnotatedAPI2.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) [2016-2017] Payara Foundation and/or its affiliates. All rights reserved.
```

Comment:
Looks like there are some copyright headers that need fixing, then this is good to go IMO